pyproject.toml

```diff
@@ -54,12 +53,12 @@ tox = "^3.25.0"
 coverage = "^6.2"
 mypy = ">= 0.920, <= 0.961"
 autopep8 = "^1.6.0"
-isort = { version = "^5.10.0", python = ">= 3.6.1" }
+isort = { version = "^5.10.0", python = ">= 3.7" }
```

Comment:
:x: the python constraint must be removed entirely

Reply:
done